### PR TITLE
Eilenberg-Moore adjunction

### PR DESCRIPTION
**Pull request by anuyts**
Builds on #764.
Result: `FreeEMAlgebra -| ForgetEMAlgebra`.

### CODE REVIEW

File: Cubical/Categories/Instances/EilenbergMoore.agda [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  ForgetFreeEMAlgebra : funcComp ForgetEMAlgebra FreeEMAlgebra ≡ M
+  ForgetFreeEMAlgebra = Functor≡ (λ x → refl) (λ f → refl)
+
+  emCounit : NatTrans (funcComp FreeEMAlgebra ForgetEMAlgebra) (funcId EMCategory)
+  carrierHom (N-ob emCounit (algebra A α , isEMA)) = α
+  strHom (N-ob emCounit (algebra A α , isEMA)) = str-μ isEMA
+  N-hom emCounit {algebra A α , isEMA} {algebra B β , isEMB} (algebraHom f isalgF) =
+    AlgebraHom≡ M (sym (isalgF))
+
+  {-
```
Comment (mortberg) [inline Cubical/Categories/Instances/EilenbergMoore.agda:89]: What is this commented code?

File: Cubical/Categories/Instances/EilenbergMoore.agda [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    ((F-hom M (N-ob η a) C.⋆ N-ob μ a) C.⋆ f)
+      ≡⟨ cong (C._⋆ f) (funExt⁻ (congP (λ i → N-ob) idr-μ) a) ⟩
+    (C.id C.⋆ f)
+      ≡⟨ C.⋆IdL f ⟩
+    f ∎
+    )
+
+  emAdjunction : FreeEMAlgebra ⊣ ForgetEMAlgebra
+  adjIso emAdjunction {a} {algebra b β , isEMB} = emBijection a (algebra b β , isEMB)
+  adjNatInD emAdjunction {a} {algebra b β , isEMB} {algebra c γ , isEMC} (algebraHom f isalgF) (algebraHom g isalgG) =
```
Comment (mortberg) [inline Cubical/Categories/Instances/EilenbergMoore.agda:154]: Long line

**Review by mortberg — CHANGES_REQUESTED** [review]
I think I only spotted one more long line and a comment that could be removed. Apart from this I'd be happy to merge once https://github.com/agda/cubical/pull/764 is merged

File: Cubical/Categories/Instances/EilenbergMoore.agda (hunk 1 above)
Comment (anuyts) [reply Cubical/Categories/Instances/EilenbergMoore.agda]: Ah sorry, a failed attempt to prove the other formulation of adjointness.

**Comment by mortberg** [comment]
And now there's some conflicts. Please fix and I'll merge

File: Cubical/Categories/Constructions/FullSubcategory.agda [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+open import Cubical.Foundations.Function
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+
+private
+  variable
+    ℓC ℓC' ℓP : Level
+
+--open Category
```
Comment (mortberg) [inline Cubical/Categories/Constructions/FullSubcategory.agda:15]: This comment can be removed

**Review by mortberg — CHANGES_REQUESTED** [review]
one small comment and some conflicts before I can merge

**Pull request merged**